server/index.js

```diff
@@ -152,37 +137,36 @@ app.post('/comment/:id', async (req, res) => {
 
     const storedComment = await db.saveComment(newComment);
 
-    res.status(201).send({
+    return res.status(201).send({
```

Comment:
Just curious what the purpose of this return is? Is it conventional, or do you need this to get your code to run?

Reply:
I suppose this is a convention, as it was flagged by airbnb style lint